tests/libtransmission/json-test.cc

```diff
@@ -121,6 +121,27 @@ TEST_P(JSONTest, testUtf8)
     tr_variantClear(&top);
 }
 
+TEST_P(JSONTest, testUtf16Surrogates)
+{
+    constexpr auto thinking_face_emoji_utf8 = "\xf0\x9f\xa4\x94"sv;
```

Comment:
Please follow .clang-tidy rule `{ key: readability-identifier-naming.ConstexprVariableCase,    value: CamelCase  }`

```suggestion
    static auto constexpr ThinkingFaceEmojiUtf8 = "\xf0\x9f\xa4\x94"sv;
```